components/lib/utils/ObjectUtils.js

```diff
@@ -416,4 +416,25 @@ export default class ObjectUtils {
 
         return [];
     }
+
+    static mutateObjectProperty(prop, value, obj) {
```

Comment:
```suggestion
    static mutateFieldData(data, field, value) {
```

Would keep the signature and naming consistent with this class?

Reply:
Makes sensee